### DISCUSSION TITLE
为什么我web保存word和pdf他会自动打开文件，我只要他下载文件怎么做

### DISCUSSION THREAD

**Discussion by lqp518** [tw93/Pake#529] [Q&A]
为什么我web保存word和pdf他会自动打开文件，我只要他下载文件怎么做，他现在打开后word是乱码，不会使用的用户也返回不了之前的页面，要用返回键才能返回，，我只在保存文件就可以。为什么他还会去打开，正常在web下他只会下载，打包后为什么会去打开word和pdf文件

**Comment by tw93** [comment]
有没有具体的页面，我看看

**Reply by lqp518** [reply]
exportToPDF(data) {
    // 创建一个新的 jsPDF 实例，并设置UTF8编码
      const doc = new jsPDF()
      addfont(doc)
      doc.addFont('bolds', 'b', 'normal')
      // 使用字体时，使用这句即可
      doc.setFont('b')
      doc.text(data, 10, 10)
      // 保存 PDF 文件并触发下载
      // doc.output('save', 'output.pdf')
      doc.save('output.pdf')

    },
    exportToWord(data) {
    // 创建一个 Blob 对象
      const blob = new Blob([data], { type: 'application/msword' })
      // 使用 FileSaver.js 保存文件并触发下载
      saveAs(blob, 'output.doc', { download: true })
    }, 传不了图片，我就是这样从网页的文件导出pdf和word 他会自动打下文件

**Reply by lqp518** [reply]
打开后普通用户也不知道怎么返回以前的页面，只能退出重新开启软件

**Reply by lqp518** [reply]
文件是下载成功了。但是他在软件里打开了文件，不会返回的用户就只能退出 重新打开软件了

**Comment by lqp518** [comment]
[tw93] 帅哥能帮看下是什么情况吗
